pkgs/tools/package-management/protonup/default.nix

```diff
@@ -0,0 +1,29 @@
+{ lib, buildPythonPackage, pythonOlder, fetchPypi, requests, configparser, pytest }:
```

Comment:
```suggestion
{ lib, buildPythonPackage, pythonOlder, fetchPypi, requests, configparser }:
```

Reply:
```bash
nix run -f https://github.com/jtojnar/nixpkgs-hammering/archive/master.tar.gz -c nixpkgs-hammer [pkg_name]
```

can help to identify issue. This is what the bot [did](https://github.com/NixOS/nixpkgs/pull/135948#issuecomment-907534081).